java/connector-node/risingwave-sink-es-7/src/main/java/com/risingwave/connector/RestHighLevelClientAdapter.java

```diff
@@ -17,140 +17,7 @@
 package com.risingwave.connector;
 
 import java.io.IOException;
-import org.apache.http.HttpHost;
-import org.apache.http.auth.AuthScope;
-import org.apache.http.auth.UsernamePasswordCredentials;
-import org.apache.http.client.CredentialsProvider;
-import org.apache.http.impl.client.BasicCredentialsProvider;
-import org.elasticsearch.client.RestHighLevelClientBuilder;
-import org.opensearch.action.bulk.BulkRequest;
-import org.opensearch.action.bulk.BulkResponse;
-import org.opensearch.client.Cancellable;
-import org.opensearch.client.RequestOptions;
-import org.opensearch.client.RestClient;
-import org.opensearch.client.RestClientBuilder;
-import org.opensearch.client.RestHighLevelClient;
-import org.opensearch.core.action.ActionListener;
 
 public interface RestHighLevelClientAdapter {
```

Comment:
The interface has only a close method remaining. Do we really need this interface? I didn't see any call on this method by the interface, and it seems that the method is only called from the concrete class. We'd better directly implement `AutoClosable` for the two clients.